code/game/mecha/equipment/tools/other_tools.dm

```diff
@@ -9,14 +9,15 @@
 	icon_state = "mecha_teleport"
 	origin_tech = "bluespace=10"
 	equip_cooldown = 150
-	energy_drain = 1000
+	energy_drain = 8000
```

Comment:
Do we want the teleporters on serpahs, and dark gygaxes, to work differently from the ones that crew can build? I thought about it, and wasn't sure. If we do, maybe the ones on seraph/darkgygax mechs should be more accurate also.